pyiron_atomistics/atomistics/structure/analyse.py

```diff
@@ -569,3 +569,90 @@ def get_voronoi_vertices(self, epsilon=2.5e-4, distance_threshold=0, width_buffe
         return xx-epsilon
+
+    def get_equivalent_points(
+        self,
+        points,
+        use_magmoms=False,
+        use_elements=True,
+        symprec=1e-5,
+        angle_tolerance=-1.0,
+        return_unique=True,
+        epsilon=1.0e-8
+    ):
+        """
+
+        Args:
+            points (list/ndarray): 3d vector
+            use_magmoms (bool): cf. get_symmetry()
+            use_elements (bool): cf. get_symmetry()
+            symprec (float): cf. get_symmetry()
+            angle_tolerance (float): cf. get_symmetry()
+            return_unique (bool): Return only points which appear once.
+            epsilon (float): displacement to add to avoid wrapping of atoms at borders
+
+        Returns:
+            (ndarray): array of equivalent points with respect to box symmetries
+        """
+        symmetry_operations = self._structure.get_symmetry(
+            use_magmoms=use_magmoms,
+            use_elements=use_elements,
+            symprec=symprec,
+            angle_tolerance=angle_tolerance
+        )
+        R = symmetry_operations['rotations']
+        t = symmetry_operations['translations']
+        x = np.einsum('jk,nj->nk', np.linalg.inv(self._structure.cell), np.atleast_2d(points))
+        x = np.einsum('nxy,my->mnx', R, x)+t
+        if any(self._structure.pbc):
+            x[:,:,self._structure.pbc] -= np.floor(x[:,:,self._structure.pbc]+epsilon)
+        if not return_unique:
+            return np.einsum(
+                'ji,mnj->mni', self._structure.cell, x
+            ).reshape((len(R),)+np.asarray(points).shape)
+        x = x.reshape(-1, 3)
+        _, indices = np.unique(
+            np.round(x, decimals=int(-np.log10(symprec))), return_index=True, axis=0
+        )
+        return np.einsum('ji,mj->mi', self._structure.cell, x[indices])
+
+    def group_points_by_symmetry(
+        self,
+        points,
+        use_magmoms=False,
+        use_elements=True,
+        symprec=1e-5,
+        angle_tolerance=-1.0,
+        epsilon=1.0e-8
+    ):
+        """
+
+        Group points according to the box symmetries
+
+        Args:
+            points (list/ndarray): 3d vector
+            use_magmoms (bool): cf. get_symmetry()
+            use_elements (bool): cf. get_symmetry()
+            symprec (float): cf. get_symmetry()
+            angle_tolerance (float): cf. get_symmetry()
+            epsilon (float): displacement to add to avoid wrapping of atoms at borders
+
+        Returns:
+            (ndarray): array of ID's according to their groups
+        """
+        all_points = self.get_equivalent_points(
+            points=points,
+            use_magmoms=use_magmoms,
+            use_elements=use_elements,
+            symprec=symprec,
+            angle_tolerance=angle_tolerance,
+            epsilon=epsilon,
+            return_unique=False
+        )
+        decimals = int(-np.log10(symprec))
+        _, inverse = np.unique(
```

Comment:
I honestly don't understand what this does, or how it relates to the box symmetry.